File: igor/Problema0209/problema0209.js
const prompt = require("prompt-sync")({ sigint: true });

var nTurma = prompt("Escreve numero turma: ")
var reprovado = 0

while( nTurma != "*") {

    nAula = parseInt(prompt("Escreve numero aula: "));
    
    nAluno = parseInt(prompt("Escreve numero aluno: "));

    for(var i =0; i<nAluno;i++){
        
        nFaltas = parseInt(prompt("Escreve numero faltas: "));

        var percentualFalta =(nFaltas*100)/nAula;

        if( percentualFalta > 25){
            reprovado = parseInt(reprovado+1);
        }
    }

    console.log(nTurma+": Numero reprovados: "+reprovado);
    var nTurma = prompt("Escreve numero turma: ")
   } 

File: igor/Problema0209/problema0209.py
nTurma = input("Escreve numero turma: ")

reprovado = 0
while nTurma!="*":
    nAula = int(input("Numero aula: "))
    nAluno = int(input("Numero aluno: "))

    for x in range(nAluno):
        nFaltas = int(input("Numero faltas: "))
        percentualFaltas = (nFaltas*100)/nAula

        if percentualFaltas > 25:
            reprovado = reprovado+1

    print(nTurma,": Numero reprovado: ",reprovado)

    nTurma = input("Escreve numero turma: ")